runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterPlugin.java

```diff
@@ -219,25 +233,33 @@ String censorMessage(final String message)
 	void updateFilteredPatterns()
 	{
 		filteredPatterns.clear();
+		filteredNamePatterns.clear();
 
 		Text.fromCSV(config.filteredWords()).stream()
 			.map(s -> Pattern.compile(Pattern.quote(s), Pattern.CASE_INSENSITIVE))
 			.forEach(filteredPatterns::add);
 
 		NEWLINE_SPLITTER.splitToList(config.filteredRegex()).stream()
-			.map(s ->
-			{
-				try
-				{
-					return Pattern.compile(s, Pattern.CASE_INSENSITIVE);
-				}
-				catch (PatternSyntaxException ex)
-				{
-					return null;
-				}
-			})
+			.map(this::compilePattern)
 			.filter(Objects::nonNull)
 			.forEach(filteredPatterns::add);
+
+		NEWLINE_SPLITTER.splitToList(config.filteredNames()).stream()
+			.map(this::compilePattern)
+			.filter(Objects::nonNull)
+			.forEach(filteredNamePatterns::add);
+	}
+
+	private Pattern compilePattern(String pattern)
```

Comment:
This can be `static`
```suggestion
	private static Pattern compilePattern(String pattern)
```